databases/percona-toolkit/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                percona-toolkit
-version             2.2.20
+version             3.0.12
 revision            1
```

Comment:
Remove the `revision` line because it starts at 0 when increasing the version and that is the default value.